yarn-project/circuit-types/src/p2p/block_attestation.ts

```diff
@@ -37,7 +37,11 @@ export class BlockAttestation extends Gossipable {
   }
 
   override p2pMessageIdentifier(): Buffer32 {
-    return BlockAttestationHash.fromField(this.payload.archive);
+    return new BlockAttestationHash(keccak256(this.signature.toBuffer()));
+  }
+
+  public proposalP2PMessageIdentifier(): Buffer32 {
```

Comment:
Is this just because we don't want to use the hash of the signature?

I'd rather just fix that problem if so. I don't like having two p2p message identifiers.

Reply:
- change to archive getter